js/filters.js

```diff
@@ -115,9 +145,27 @@ function TypeaheadFilter(elm) {
   var key = this.$body.data('dataset');
   var dataset = typeahead.datasets[key];
   this.typeaheadFilter = new typeaheadFilter.TypeaheadFilter(this.$body, dataset);
+  this.typeaheadFilter.$body.on('change', 'input[type="checkbox"]', this.handleNestedChange.bind(this));
 }
 
 TypeaheadFilter.prototype = Object.create(Filter.prototype);
 TypeaheadFilter.constructor = TypeaheadFilter;
 
+TypeaheadFilter.prototype.handleChange = function() {};
```

Comment:
This overrides the inherited `Filter.prototype.handleChange` method, which doesn't work well for this filter subclass--the base class calls `handleChange` when any `input` changes, but in this case, we want to ignore changes on the text box and only listen on the checkboxes. Definitely open to suggestions about a better way to handle this, though.